WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -147,6 +147,9 @@
 /* No comment provided by engineer. */
 "A refund will not be issued to the customer. You will need to manually issue the refund through Stripe." = "A refund will not be issued to the customer. You will need to manually issue the refund through Stripe.";
 
+/* Message explaining that the site may have an invalid SSL certificate. */
+"A secure connection to the site could not be made. Please make sure that your site has a valid SSL certificate." = "A secure connection to the site could not be made. Please make sure that your site has a valid SSL certificate.";
+
```

Comment:
Note: the changes in that file were not commited when the CI was triggered and beta was built, due to the current bug on toolkit about strings file commit (see PR description for details).